src/pages/project/cartzoom.js

```diff
@@ -143,6 +165,17 @@ module.exports = {
         }
       });
     }
+
+    var elBounding = this.refs.bounding.getDOMNode();
+
+    // Turn off animation flag (used to block interactions)
+    elBounding.addEventListener('transitionend', (event) => {
```

Comment:
This will break on Safari, IE for the player; you will need to do detection of which `transitionend` event to listen for (alternatively, you could use a timeout). Also, this will be a 100% breaking experience for browsers that don't support CSS transitions -- are we OK with that?